swarm/api/http/server.go

```diff
@@ -591,6 +591,15 @@ func (s *Server) HandleGetFile(w http.ResponseWriter, r *Request) {
 func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	s.logDebug("HTTP %s request URL: '%s', Host: '%s', Path: '%s', Referer: '%s', Accept: '%s'", r.Method, r.RequestURI, r.URL.Host, r.URL.Path, r.Referer(), r.Header.Get("Accept"))
 
+	if r.RequestURI == "/" && strings.Contains(r.Header.Get("Accept"), "text/html") {
```

Comment:
My feedback to this solution is that if the requestor is a browser, this works fine, but in case it doesn't have the accept header set to `text/html`, e.g. a JSON or AJAX request, `/` would still return 404.

Reply:
@holisticode I think that is fine, if a user is hitting this with AJAX then they probably made a mistake in their JavaScript, similar argument for JSON requests (what would the user be expecting?).

Browsers are different as it is more likely people just try it out in the URL bar.

Reply:
My use case: Checking if a local node is alive for example. I was using this for an interface I wrote last year. If the local node was alive, I'd use that one to send requests, otherwise I would use the gateway. Of course even a `404` response is a `I am alive` response, albeit rather counter-intuitive. We may also have a `status` page sooner or later, but sending a request to `/` to check for `alive` seems a possible approach. No strong opinion though.

Reply:
@holisticode My understanding was that use case would be resolved with issue #185 ? I'm happy to address it there after this is resolved

Reply:
Oh @cooganb sorry, was not aware of https://github.com/ethersphere/go-ethereum/issues/185. I don't care much where it's being addressed, although looks like a good candidate here :) Nevertheless wherever fits best.